packaging/maven/publish-maven-release.sh

```diff
@@ -3,11 +3,11 @@ set -xeuo pipefail
 case "$CHANNEL" in
   "3.8")
     REPO_ID=ossrh
-    SOURCE_URL="https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+    SOURCE_URL="https://ossrh-staging-api.central.sonatype.com/service/local/staging/deploy/maven2/"
```

Comment:
This is the endpoint for publishing release to the maven central repository, the other is no longer used.